CHANGELOG.md

```diff
@@ -257,3 +275,11 @@ Release the first milestone
 [v0.5.0-b1]: https://github.com/teracyhq/dev/milestone/1?closed=1
 
 [v0.5.0-b2]: https://github.com/teracyhq/dev/milestone/4?closed=1
+
+[v0.5.0-c1]: https://github.com/teracyhq/dev/milestone/5?closed=1
+
+[315]: https://github.com/teracyhq/dev/issues/315
```

Comment:
Because the project base config description not clear enough, if user wants to know more they can check the issues